composer.json

```diff
@@ -22,10 +22,11 @@
     "source": "https://github.com/bryanjhv/slim-session.git"
   },
   "require": {
+    "php": ">=8",
```

Comment:
Breaks compatibility with Slim 4.x.

Reply:
Yes.

If use `#[\ReturnTypeWillChange]` in all `ReturnTypeWillChange` warning points. that could be suppress warns in PHP8.1.

I suggested another resolve pattern in PR text.
> use #[ReturnTypeWillChange] to all warned methods.

I can rewrite to `use #[ReturnTypeWillChange]` pattern.